sktime/clustering/tests/partitioning/test_center_initializers.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
```

Comment:
I suggest to change the folder structure here to: `sktime/clustering/partitioning/tests/test_center_initializers.py`, makes it easier to find the corresponding tests and is in line with the rest of the repo